### ISSUE TITLE
Review & fix 'orofacial cleft' (DOID:0050567) 'disease has location' axiom 

### ISSUE THREAD

**Issue by allenbaron** [DiseaseOntology/HumanDiseaseOntology#1124]
'orofacial cleft' (DOID:0050567) references itself as the location in the axiom `'physical disorder'
 and (('disease has basis in' some symptom)
 and ('disease has location' some 'orofacial cleft'))`, which is clearly an error.

This axiom either needs to be generalized to use an anatomical term encompassing the lip and/or palate or removed. The logical axioms on the child terms may also need to be reviewed.

**Comment by lschriml** [comment]
Agreed, there should not have been the annotation: disease has location' some 'orofacial cleft'

I have added a SubClassOf axiom:
('disease has location' some lip or 'disease has location' some 'soft palate' or 'disease has location' some 'hard palate')

**Comment by allenbaron** [comment]
Fixed by commits d361faedf7ff53e66d9f63b661ff437575a1bd7d and 02948a137194189f889bdbafe8deff2b23c415e8.
